code/modules/vending/medical.dm

```diff
@@ -28,14 +28,14 @@
 					/obj/item/sensor_device = 2,
 					/obj/item/pinpointer/crew = 2)
 	contraband = list(/obj/item/reagent_containers/pill/tox = 3,
-		              /obj/item/reagent_containers/pill/morphine = 4,
-		              /obj/item/reagent_containers/pill/multiver = 6,
-		              /obj/item/storage/box/hug/medical = 1)
+						/obj/item/reagent_containers/pill/morphine = 4,
+						/obj/item/reagent_containers/pill/multiver = 6,
+						/obj/item/storage/box/hug/medical = 1)
 	premium = list(/obj/item/reagent_containers/medigel/instabitaluri = 2,
-		           /obj/item/storage/pill_bottle/psicodine = 2,
-		           /obj/item/reagent_containers/hypospray/medipen = 3,
-		           /obj/item/storage/belt/medical = 3,
-		           /obj/item/storage/firstaid/advanced = 2)
+					/obj/item/storage/pill_bottle/psicodine = 2,
+					/obj/item/reagent_containers/hypospray/medipen = 3,
+					/obj/item/storage/belt/medical = 3,
+					/obj/item/storage/firstaid/advanced = 2)
```

Comment:
Revert this file plox

Reply:
I think it's fine changing the spaces to tabs, if tabs are what's generally used in the file.

Reply:
The file had an ugly mix of tabs and spaces, so I replaced all of them with spaces.

![image](https://user-images.githubusercontent.com/32391752/71517613-301fce00-28af-11ea-8564-a8d974e6b0e4.png)